res/container-tests/Containerfile.ubi10

```
@@ -0,0 +1,35 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

```suggestion
FROM registry.access.redhat.com/ubi9/ubi:latest
```
Should be 10 really, but it's not out yet ;-)